Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -185,6 +191,12 @@ namespace AZ
             //! Get draw filter mask
             RHI::DrawFilterMask GetDrawFilterMask() const;
 
+            using NotificationEvent = AZ::Event<>;
```

Comment:
suggestion: Could this be given a more descriptive name like `FrameNotificationEvent`?